R/models.R

```diff
@@ -190,6 +190,19 @@ predict_model.H2OModel <- function(x, newdata, type, ...){
       stop('This h2o model is not currently supported.')
   }
 }
+#' @export
+predict_model.ranger <- function(x, newdata, type, ...) {
+  if (!requireNamespace('ranger', quietly = TRUE)) {
+    stop('The ranger package is required for predicting ranger models')
+  }
+  res <- predict(x, data = newdata, ...)
+  switch(
+    type,
+    raw = as.data.frame(Response = colnames(res$predictions)[max.col(res$predictions)], stringsAsFactors = FALSE),
```

Comment:
`raw` is used for regression models to return a single column data.frame with the predicted response. This seems to return a class membership? Or maybe I don't understand rangers return format enough..?

Reply:
Is the single column of the class prediction not what is wanted?  I patterned mine after the predict_model.lda.  ranger doesn't seem to have a built in class prediction, so I just picked the class with the highest probability.

Reply:
Raw is not used by classifiers at all - only regressors

Reply:
Oh... makes sense. I'll make some adjustments.  Thanks!